cmd/pinniped-proxy/src/cli.rs

```diff
@@ -12,8 +12,16 @@ pub struct Options {
     #[structopt(
         short = "p",
         long = "port",
+        env = "PINNIPED_PROXY_PORT",
```

Comment:
Nice, I hadn't realised that the `structopt` crate also enabled env vars as alternatives for options.